www/js/controllers/database-controller.js

```diff
@@ -0,0 +1,14 @@
+module.controller('MyCtrl', function($scope, $cordovaSQLite) {
+
+  var db = $cordovaSQLite.openDB({ name: "mood_tracker_database" });
+
```

Comment:
This is example code, left for the moment so others can better understand structure.